front/src/components/stories/commonStories/MissionAcceptation.tsx

```diff
@@ -0,0 +1,21 @@
+import { Link } from "react-router-dom";
+import { BigTitle } from "../../styled/BigTitle";
```

Comment:
On peut faire du path alias via le package.json
https://arunmichaeldsouza.com/blog/aliasing-module-paths-in-node-js